ebean-ddl-generator/src/main/java/io/ebeaninternal/dbmigration/ddlgeneration/platform/ClickHouseDbArray.java

```diff
@@ -15,6 +15,7 @@ class ClickHouseDbArray {
     mapping.put("integer[]", "Array(UInt32)");
     mapping.put("bigint[]", "Array(UInt64)");
     mapping.put("float[]", "Array(Float32)");
+    mapping.put("float4[]", "Array(Float32)");
```

Comment:
There were some issues for Clickhouse. They are in commit https://github.com/ebean-orm/ebean/pull/3107/commits/81443b77bc6093f3c6f9f6fa61e4608ee362d0bb